fmn/consumer/backend.py

```diff
@@ -175,9 +178,11 @@ def read(queue_object):
                 idx = recipient['filter_id']
                 fltr = session.query(fmn.lib.models.Filter).get(idx)
                 fltr.fired(session)
-        except:
-            # If anything happens, put the message back in the queue and bail
-            ch.cancel()
+        except RuntimeError:
+            yield ch.basic_nack(delivery_tag=method.delivery_tag)
+            return
+        except smtplib.SMTPSenderRefused:
```

Comment:
Hmm is dropping email the ideal way of handling this or is there a possibility to split the email to be shorter?